econml/dml.py

```diff
@@ -393,48 +456,38 @@ class SparseLinearDMLCateEstimator(_DMLCateEstimatorBase):
         If RandomState instance, random_state is the random number generator;
         If None, the random number generator is the RandomState instance used
         by `np.random`.
-
```

Comment:
Since @moprescu has finished implementing the debiased Lasso, why isn't the debiased lasso inference part of this release??

Reply:
It hasn't gone through PR yet.